inc/classes/class-srm-redirect.php

```diff
@@ -98,6 +98,28 @@ public function match_redirect( $requested_path ) {
 			$normalized_requested_path = $requested_path;
 		}
 
+		// normalise the request path with and without query strings, for comparison later
+		$requested_query_params             = '';
+		$normalized_requested_path_no_query = false;
+
+		if ( function_exists( 'wp_parse_url' ) ) {
+			$parsed_requested_path = wp_parse_url( $normalized_requested_path );
+		} else {
+			$parsed_requested_path = parse_url( $normalized_requested_path );
+		}
+
+		if ( is_array( $parsed_requested_path ) ) {
+			$normalized_requested_path_no_query = '';
+			$requested_query_params             = '';
```

Comment:
Duplicated variables declaration.
IMO, we can safely use `$parsed_requested_path` without declaring 
intermediate variables.

Reply:
Updated.